MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -5,6 +5,15 @@
 import Foundation
 import FeedFeature
 
+struct FeedLoadErrorViewModel {
+	let message: String
+}
+
+protocol FeedLoadErrorView {
+	func display(_ viewModel: FeedLoadErrorViewModel)
+	func hideErrorView()
+}
```

Comment:
Suggestion:
It can be better to have `FeedErrorViewModel.message` as optional and hide the view by calling `display(.init(message: nil))`, because `hide` is a very generic name and can have conflicts with other methods in the future. 
Also it will keep all the state transitions centralized into a single state type.

```suggestion
struct FeedLoadErrorViewModel {
	let message: String?
}

protocol FeedLoadErrorView {
	func display(_ viewModel: FeedLoadErrorViewModel)
}
```